src/contexts/filterContext/context.js

```diff
@@ -0,0 +1,46 @@
+import { useContext, useReducer, useEffect, createContext } from "react";
+import { reducerFunction } from "./reducer";
```

Comment:
it would be better to name this as filterReducer

Reply:
Thanks for pointing it out will rename it